cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/ObjectMappedTableProperties.java

```diff
@@ -40,22 +41,53 @@
   /**
    * The type of object in the table.
    */
-  public static final String TYPE = "type";
+  public static final String OBJECT_TYPE = "object.type";
 
   /**
    * The schema of objects in the table.
    */
-  public static final String SCHEMA = "schema";
+  public static final String OBJECT_SCHEMA = "object.schema";
+
+  /**
+   * The name of the key to use when exploring the table. Defaults to "key".
+   */
+  public static final String EXPLORE_KEY_NAME = "explore.key.name";
+
+  /**
+   * The type of the key when exploring the table. Defaults to bytes.
+   */
+  public static final String EXPLORE_KEY_TYPE = "explore.key.type";
```

Comment:
I'll add more to the javadocs.  Our Schema class does not let you have 2 fields that have the same name, so I will document that as well.